src/main/java/io/blamer/bot/conversation/routes/Registry.java

```diff
@@ -61,32 +64,28 @@ public class Registry implements Conversation {
   @Value("${answers.registry.description}")
   private String description;
 
-  /**
-   * The requester.
-   */
-  private final RSocketRequester requester;
+  private final WebClient webClient;
 
   @Override
-  public SendMessage messageFromUpdate(final Update update) {
+  public Mono<SendMessage> messageFromUpdate(final Update update) {
     final String chat = String.valueOf(update.getMessage().getChatId());
-    try {
-      final String token = update.getMessage().getText()
-        .replace("/registry", " ")
-        .strip();
-      return this.requester
-        .route("hub.auth")
-        .data(new RegistryRequest(token, chat))
-        .retrieveMono(RegistryResponse.class)
-        .map(auth -> new SendMessage(auth.getChat(), auth.getText()))
-        .block();
-    } catch (final RuntimeException ex) {
-      final SendMessage error = new SendMessage(
-        chat,
-        "`%s`".formatted(ex.getMessage())
-      );
-      error.enableMarkdown(true);
-      return error;
-    }
+    final String token = update.getMessage().getText()
+      .replace("/registry", " ")
+      .strip();
+    return this.webClient.post()
+      .uri("/api/v1/hub/auth")
+      .contentType(MediaType.APPLICATION_JSON)
+      .bodyValue(new RegistryRequest(token, chat))
+      .retrieve()
+      .bodyToMono(RegistryResponse.class)
+      .doOnError(
+        err -> new SendMessage(
+          chat,
+          "`%s`".formatted(err.getMessage())
```

Comment:
@l3r8yJ markdown enabling is missing here

Reply:
as far as i understand `` symbols work here as md

Reply:
@l3r8yJ to my knowledge, in telegram it will be represented just with `` but not in markdown